src/StackHCI/help/Get-VMAzStackHCIAttestation.md

```diff
@@ -0,0 +1,75 @@
+---
+external help file: Az.StackHCI-help.xml
+Module Name: Az.StackHCI
+online version: https://docs.microsoft.com/powershell/module/az.stackhci/get-vmazstackhciattestation
+schema: 2.0.0
+---
+
+# Get-VMAzStackHCIAttestation
+
+## SYNOPSIS
+Get-VMAzStackHCIAttestation shows a list of guests added to IMDS Attestation on a node.
+
+## SYNTAX
+
+```
+Get-VMAzStackHCIAttestation [-Local] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get-VMAzStackHCIAttestation shows a list of guests added to IMDS Attestation on a node.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```powershell
+C:\PS\>Get-VMAzStackHCIAttestation
+```
+
+Get all guests on cluster.
+
+### EXAMPLE 2
+```powershell
+C:\PS\>Get-VMAzStackHCIAttestation -Local
+```
+
+Get all guests on current node.
+
+### EXAMPLE 3
+```powershell
+C:\PS\>Invoke-Command -ScriptBlock {Get-VMAzStackHCIAttestation} -ComputerName "node1"
```

Comment:
Why there is `Invoke-Command`?

Reply:
This particulary one is an inccorect leftover from a previous version of the cmdlet (that could not be run except locally). Pushing a few updates shortly